pkg/js-search/rollup.config.js

```diff
@@ -0,0 +1,38 @@
+import babel from "rollup-plugin-babel";
+import { terser } from "rollup-plugin-terser";
+
+const emitModulePackageFile = () => {
+  return {
+    name: "emit-module-package-file",
+    generateBundle() {
+      this.emitFile({
+        type: "asset",
+        fileName: "package.json",
+        source: `{"type":"module"}`
+      });
+    }
+  };
+};
```

Comment:
I would prefer if we used Vite for this (for consistencies sake): as Vite already uses rollup under the hook (among other things), the transition should be almost seamless.

Reply:
Vite is now used to build (and also, building is now possible).